CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- PercentileHistoricComponent and HistoricComponent now automatically choose a sensible dropdown
```

Comment:
Would have preferred something more specific than "sensible", i.e., explicitly match the default from the API.